src/RBF/rbf.cpp

```diff
@@ -1181,7 +1381,105 @@ double RBF::calcDist(int i, int j)
  */
 double RBF::calcDist(const std::array<double,3>& point, int j)
 {
-    return norm2(point-m_node[j]);
+    return norm2(point - m_node[j]);
+}
+
+/*!
+ * Initialize activation of monomials terms of linear polynomial part 
+ * If all the nodes are aligned on a plane normal to a cartesian coordinate 
+ * the related coordinate if disable in the polynomial function (no dependency 
+ * can be found).
+ */
+void RBF::initializePolynomialActiveBasis()
+{
+    std::vector<int> activeSet(RBFKernel::getActiveSet());
+
+    // Initialize active terms for an only constant linear polynomial
+    m_polyActiveBasis.clear();
+    m_polyActiveBasis.insert(0);
```

Comment:
The documentation says "Active terms of linear polynomial, -1 is constant, 0,1,2 the system coordinates", it seems to me that the documentation should be updated.